app/uk/gov/hmrc/hmrcemailrenderer/templates/itr/itrEmailVerification.scala.html

```diff
@@ -0,0 +1,8 @@
+@(params: Map[String, Any])
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, "Verify your email address") {
+<p style="margin: 0 0 30px; font-size: 19px;">You’re getting this email because you’re applying to use a venture capital scheme.
+    We need you to confirm your email address so you can use the service.</p>
+<p style="margin: 0 0 30px; font-size: 19px;">To verify your email address click the link below:</p>
+<p style="margin: 0 0 30px; font-size: 19px;"><a href="@{params("verificationLink")}" style="color: #005EA5;">@{params("verificationLink")}</a></p>
+<p style="margin: 0 0 30px; font-size: 19px;">From HMRC</p>
```

Comment:
Hi @jakobgrunig 
That is still under discussion. I may need to update the template with "h4>This is a test email only. Should you receive this email please ignore it</h4>"